Assets/MixedRealityToolkit/UX/README.md

```diff
@@ -2,76 +2,34 @@
- 
-**Disclaimer**: The default value of any font may be effected by the texture size of that font or how the font was imported into Unity. These tests were performed based on the default Arial font in Unity, as well as one other imported font.
-
-![Font size with scaling factors](../../../External/ReadMeImages/TextPrefabInstructions07.png)
-
-### [Scripts](Scripts)
-
-
-
+## Examples - UI controls and building blocks
+|  [![Bounding Box and App Bar](/External/ReadMeImages/MRTK_AppBar_BoundingBox.jpg)](/Assets/MixedRealityToolkit-Examples/UX/Readme/README_BoundingBoxGizmoExample.md) [Bounding Box and App Bar](/Assets/MixedRealityToolkit-Examples/UX/Readme/README_BoundingBoxGizmoExample.md) | [![Interactable Objects](/External/ReadMeImages/MRTK_InteractableObject_HolographicButton.jpg)](/Assets/MixedRealityToolkit-Examples/UX/Readme/README_InteractableObjectExample.md) [Interactable Objects](/Assets/MixedRealityToolkit-Examples/UX/Readme/README_InteractableObjectExample.md) | [![Object Collection](/External/ReadMeImages/MRTK_ObjectCollection.jpg)](/Assets/MixedRealityToolkit-Examples/UX/Readme/README_ObjectCollection.md) [Object Collection](/Assets/MixedRealityToolkit-Examples/UX/Readme/README_ObjectCollection.md) |[![Keyboard](/External/ReadMeImages/MRTK_Keyboard.jpg)](/Assets/MixedRealityToolkit-Examples) [Keyboard](/Assets/MixedRealityToolkit-Examples) |
+|:--- | :--- | :--- | :--- |
+| Standard UI for manipulating objects in 3D space | Modular script for making objects interactable with visual states, including Holographic button  | Script for lay out an array of objects in a three-dimensional shape | System style keyboard prefab for Unity projects |  
+| ![Dialog and Progress](/External/ReadMeImages/MRTK_DialogProgress.jpg) [Dialog](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/Dev_Working_Branch/Assets/MixedRealityToolkit-Examples/UX/Readme/README_DialogsExample.md) / [Progress](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/Dev_Working_Branch/Assets/MixedRealityToolkit-Examples/UX/Readme/README_ProgressExample.md) | [![Solver system](/External/ReadMeImages/MRTK_SolverSystem.jpg)](/Assets/MixedRealityToolkit-Examples/Utilities/Readme/README_SolverSystem.md) [Solver system](/Assets/MixedRealityToolkit-Examples/Utilities/Readme/README_SolverSystem.md) | [![Interactive button demos](/External/ReadMeImages/MRTK_InteractiveButtons.jpg)](/Assets/MixedRealityToolkit-Examples/UX/Readme/README_InteractiveButtonComponents.md) [Interactive button demos](/Assets/MixedRealityToolkit-Examples/UX/Readme/README_InteractiveButtonComponents.md) | [![ToolTip & Label](/External/ReadMeImages/MRTK_Tooltip.jpg)](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/Dev_Working_Branch/Assets/MixedRealityToolkit-Examples/UX/Readme/README_TooltipExample.md) [ToolTip & Label](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/Dev_Working_Branch/Assets/MixedRealityToolkit-Examples/UX/Readme/README_TooltipExample.md) |
```

Comment:
Let's make sure we're using the shorter uri links instead of the fully qualified path.

for example: 
```
Assets/MixedRealityToolkit-Examples/UX/Readme/README_DialogsExample.md
```
Instead of:
```
https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/Dev_Working_Branch/Assets/MixedRealityToolkit-Examples/UX/Readme/README_DialogsExample.md
```

Reply:
Updated, thanks!